src/components/layout/Footer.js

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+
+var style = {
```

Comment:
@koitoror Style this component with an external css file. You can assign the divs classNames

Reply:
Done